.github/workflows/ci.yml

```diff
@@ -13,53 +13,51 @@ jobs:
     name: Build and test
     runs-on: ubuntu-18.04
     env:
-      CODECOV_TOKEN: "${{ secrets.CODECOV_TOKEN }}"
-
+      CODECOV_TOKEN: '${{ secrets.CODECOV_TOKEN }}'
     strategy:
       matrix:
         node-version: [12.x]
 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        # pulls all commits (needed for lerna / semantic release to correctly version)
-        fetch-depth: "0"
-
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-
-    - name: Cache npm
-      uses: actions/cache@v1
-      with:
-        path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
-        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ${{ runner.OS }}-node-
-          ${{ runner.OS }}-
-
-    - name: Get yarn cache directory path
-      id: yarn-cache-dir-path
-      run: echo "::set-output name=dir::$(yarn cache dir)"
-    - name: Cache yarn
-      uses: actions/cache@v1
-      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
-      with:
-        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-yarn-
-
-    - name: Install dependencies
-      run: yarn install --frozen-lockfile
-    - name: Build packages
-      run: yarn build
-    - name: Run ESLint
-      run: yarn lint --quiet
-    - name: Run unit tests
-      run: yarn test
-
-    - name: Report code coverage
-      run: curl -s https://codecov.io/bash | bash
-
+      - uses: actions/checkout@v2
```

Comment:
nothing should have changed here, prettier applied this formatting.

Reply:
![image](https://user-images.githubusercontent.com/812905/100958214-c6517080-34d0-11eb-8b1d-a7fdb0ed7df6.png)
^one of my favorite github features